scripts/superdesk/editor/editor.js

```diff
@@ -219,6 +228,12 @@ function EditorService(spellcheck, $rootScope, $timeout, $q) {
             return true;
         }
 
+        // ignore editing keys
+        if (self.EDITING[event.keyCode]) {
+            event.stopPropagation();
+            return true;
```

Comment:
I think it shouldn't be here, because this `shouldIgnore` returns true when there is no change to content, but if you press enter or backspace there is a change so it should return false and trigger autosave..
probably better would be to move this outside this function right into event handler..